docs/_includes/image-placeholder.html

```diff
@@ -28,15 +28,15 @@ <h3>Variable widths with borders</h3>
             </div>
             <div>
                 <svg aria-label="image not found" class="image-placeholder" focusable="false"
-                    style="border: 1px solid black; height: 150px; width: 150px"
+                    style="border: 1px solid orange; height: 150px; width: 150px"
                     role="img">
                     {% include symbol.html name="placeholder" prefix="image" %}
                 </svg>
                 </svg>
             </div>
             <div>
                 <svg aria-label="image not found" class="image-placeholder" focusable="false"
-                    style="border: 1px dotted black; height: 250px; width: 250px" role="img">
+                    style="border: 1px dotted orange; height: 250px; width: 250px" role="img">
```

Comment:
I don't know how far we want to take this, but it looks like this change appears to create a color contrast issue on light mode. The dotted orange seems too faint. 
<img width="699" alt="image" src="https://github.com/eBay/skin/assets/1675667/e2568933-e3e1-4e25-8479-6e4e5b487e30">

Reply:
We might need to pick a color that improves the contrast then. Can make that change.